index.d.ts

```diff
@@ -52,6 +52,11 @@ declare namespace contextMenu {
 		*/
 		readonly copyImageAddress?: string;
 
+		/**
+		@default 'Copy Image'
+		*/
+		readonly copyImage?: string;
```

Comment:
Not the correct order. The order here should follow the order shown in the menu.